resources/build-integration.md

```diff
@@ -21,6 +21,51 @@ We recommend checking out the READMEs of our other integration repositories, lik
 
 All integration repositories must contain a file titled `CONTRIBUTING.md`. The template is [here](/templates/CONTRIBUTING.md) and should be filled according to the details of your repository..
 
+### 3.1 Docker & Docker-compose
+
+The integration should be ready to use with `Docker`, and it must contain a `Docker Compose` file configurated following the patterns of the other repositories.
+- Must have a service `package` and a `meilisearch` service.
+  - Must depend and contain a link to `meilisearch` service.
```

Comment:
Not sure to understand this sentence correctly. Does that mean a link to the official docker image of Meilisearch?

Reply:
That's the official docker link: https://hub.docker.com/r/getmeili/meilisearch not something like `myrandomusername/meilisearch`.